crates/parameters/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+edition = "2021"
+homepage = "https://github.com/hulks/hulk"
+license = "GPL-3.0-only"
+name = "parameters"
+version = "0.1.0"
+
+[dependencies]
+color-eyre = {workspace = true}
+log = {workspace = true}
+serde = {workspace = true}
+serde_json = {workspace = true}
+serialize_hierarchy = {workspace = true}
+thiserror = {workspace = true}
+tokio = {workspace = true}
```

Comment:
Please fix your editor that you always have a newline at the end of each line according to POSIX standard.

Reply:
Fixed